content/blog/2021/11/2021-11-12-interview-duchess-hacktoberfest.adoc

```diff
@@ -0,0 +1,157 @@
+
+So I imagined a governess, very confident, efficient, very clever, scrupulously combed, nicely dressed, who might represent the personality of what Jenkins is made of.
+
+=== Do you have anything to add?
+This time I was able to participate through Pull Requests related to translation and drawing. I’ll probably do it again!
+
+=== Bertha Torres contributions during hacktoberfest
+* https://github.com/jenkins-infra/jenkins.io/pull/4627
+* https://github.com/jenkinsci/jenkins/pull/5796
+* https://github.com/jenkinsci/jenkins/pull/5797
+* https://github.com/jenkinsci/workflow-job-plugin/pull/214
+* https://github.com/jenkinsci/workflow-job-plugin/pull/215
+* https://github.com/jenkinsci/ant-plugin/pull/78
+* https://github.com/jenkinsci/ant-plugin/pull/77
+
+== Pauline Iogna’s Interview
+
+image:/images/post-images/2021-11-duchess-hacktoberfest/pl.jpg[Pauline Iogna picture, height=200]
+
+* link:https://twitter.com/pauline_io[Twitter: @pauline_io]
+
+=== Hello Pauline, can you introduce yourself?
+I’m a backend developer in Java/Scala, and an active member of Duchess France.
+
+=== What did you think about Open Source, and maybe Jenkins, before starting the hackathon?
+I didn't really have an opinion about Jenkins before doing the hackathon.
+Like many other people, I’m a Jenkins user but I didn’t know its source code.
+
+=== What motivated you to participate in this hackathon?
+Actually participating in Hacktoberfest.
+Just bringing a contribution, even modest, on an open source project.
+
+=== What did you discover through that hackathon?
+Technically, I discovered how to do MVC with Apache Jelly.
+I also rediscovered the contribution process of Open Source projects.
+
+The Hacktoberfest mechanism is really well thought out for onboarding people wanting to start working on Open Source projects.
+With Angélique’s help on top of that to animate sessions on the Duchess France Slack channel, we had the best conditions to contribute to that important project.
+
+=== What will you remember from this experience?
+It’s not that simple to contribute to a project you’re just discovering, it requires a bit of investment and patience.
+On Jenkins, the contributors are really reactive and the Pull Requests are quickly reviewed.
+The feedback is useful and benevolent. 
+There is one Jira project listing all the features and bugs to be worked on for the project.
+Some tickets are flagged as “newbie” allowing those who are beginners to pick easily and quickly doable tickets.
+
+=== Pauline Iogna contributions
+* https://github.com/jenkinsci/jenkins/pull/5835
+* https://github.com/jenkinsci/jenkins/pull/5793
+
+== A few words about the organization within Duchess France
+The framework was intentionally simple and flexible. We started with an link:https://www.youtube.com/watch?v=Cwlv2J9U-qs [online kick start meeting for describing a bit of context], and more precisely the Jenkins universe.
+
```

Comment:
Let's fix the broken link to the video and embed the video player into the page.

```suggestion
The framework was intentionally simple and flexible. We started with an link:https://www.youtube.com/watch?v=Cwlv2J9U-qs[online kick start meeting for describing a bit of context], and more precisely the Jenkins universe.

.Online kick start meeting for context
video::Cwlv2J9U-qs[youtube, width=852, height=480]

```